### PR TITLE
feat: add new session template

### PR DESCRIPTION
**Pull request by Eomm**
Hello,
I'm adding a new template to host SH Session

Here is the discussion we took:

- https://discord.com/channels/963080048380895352/1150689772029878305
- https://gist.github.com/Eomm/77ad35294a0c09d4049371c572304c56

Here is a preview of this PR:

https://github.com/Eomm/sh-sessions/issues/new/choose

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/new-session.yaml [hunk 1]
```diff
@@ -0,0 +1,101 @@
+name: 🤝 Organize a new Schrödinger Hat session
+description: Create a cell in your city by organizing a Schrödinger Hat Session
+body:
+  - type: markdown
+    attributes:
+      value: |
+        # Guidelines
+
+        Organizing a tech event is not easy, but having a community to ask for advice and help makes everything possible.  
+        Schrödinger Hat allows you to create a CELL in your city by organizing a Schrödinger Session.
```
Comment (nicpuppa) [inline .github/ISSUE_TEMPLATE/new-session.yaml]:
```suggestion
        Schrödinger Hat helps you to create a Schrödinger Session in your city.
```

File: .github/ISSUE_TEMPLATE/new-session.yaml [hunk 2]
```diff
@@ -0,0 +1,101 @@
+name: 🤝 Organize a new Schrödinger Hat session
+description: Create a cell in your city by organizing a Schrödinger Hat Session
```
Comment (nicpuppa) [inline .github/ISSUE_TEMPLATE/new-session.yaml]:
```suggestion
description: Organize a Schrödinger Session in your city
```

I don't like here the word `cell`

File: .github/ISSUE_TEMPLATE/new-session.yaml [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+        Organizing a tech event is not easy, but having a community to ask for advice and help makes everything possible.  
+        Schrödinger Hat allows you to create a CELL in your city by organizing a Schrödinger Session.
+
+        Schrödinger Hat provides:
+        - Its communication channels.
+        - Its event management tools (Meetup, Eventbrite, etc.).
+        - Its experience and contacts with Community Partners.
+
+        This form is designed to assist you in organizing a new Schrödinger Hat Session in your city!
```
Comment (nicpuppa) [inline .github/ISSUE_TEMPLATE/new-session.yaml]:
```suggestion
        This form is designed to assist you in organizing a new Schrödinger Session in your city!
```

File: .github/ISSUE_TEMPLATE/new-session.yaml [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+    id: description
+    attributes:
+      label: Description
+      placeholder: Describe the session that is scheduled to take place.
+    validations:
+      required: true
+  - id: requirements
+    type: checkboxes
+    attributes:
+      label: To organize a Schrödinger Hat Session
```
Comment (nicpuppa) [inline .github/ISSUE_TEMPLATE/new-session.yaml]:
```suggestion
      label: To organize a Schrödinger Session 
```

File: .github/ISSUE_TEMPLATE/new-session.yaml [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+      options:
+        - label: The event will follow the Schrödinger Hat [Code of Conduct](https://github.com/Schrodinger-Hat/sh-sessions/blob/main/CODE_OF_CONDUCT.md)
+          required: true
+        - label: The event must be able to accommodate at least 30 people
+          required: true
+        - label: The event must host at least 2 speakers
+          required: true
+        - label: Talks must be given in English
+          required: true
+        - label: The event must be streamed on Schrödinger Session's YouTube channel
```
Comment (nicpuppa) [inline .github/ISSUE_TEMPLATE/new-session.yaml]:
```suggestion
        - label: The event must be streamed on Schrödinger Hat's YouTube channel
```

**Review by nicpuppa — COMMENTED** [review]
Thanks a lot @Eomm for your great work ❤️ 

I left some suggestions, please have a look 👀

File: .github/ISSUE_TEMPLATE/new-session.yaml [hunk 6]
```diff
@@ -0,0 +1,101 @@
+name: 🤝 Organize a new Schrödinger Hat session
```
Comment (nicpuppa) [inline .github/ISSUE_TEMPLATE/new-session.yaml]:
```suggestion
name: 🤝 Organize a new Schrödinger Session
```

**Review by nicpuppa — APPROVED** [review]
LGTM 🚀 

Let's wait another approval before merging this, thanks again 💪

**Review by TheJoin95 — APPROVED** [review]

**Pull request merged**